src/main/java/org/terasology/launcher/game/VersionHistory.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.launcher.game;
+
+import com.vdurmont.semver4j.Requirement;
+import com.vdurmont.semver4j.Semver;
+import org.terasology.launcher.model.GameIdentifier;
+
+/**
+ * Terasology versions which introduce specific features.
+ * <p>
+ * These features change the way Launcher must interact with Terasology.
+ */
+public enum VersionHistory {
+    /**
+     * The preview release of v4.1.0-rc.1 is the first release with LWJGL v3.
+     * See https://github.com/MovingBlocks/Terasology/releases/tag/v4.1.0-rc.1
+     */
+    LWJGL3("[4.1.0,)"),
+    PICOCLI("[5.1.0,)");
```

Comment:
Also asked this on Discord: we've got a bunch of preview builds with `engine-5.1.0-SNAPSHOT` that don't use POSIX syntax yet - those would stop working, is that correct?

Reply:
I've updated PICOCLI to start at 5.2.0-SNAPSHOT.